storage/src/include/custom_part_lib.ycp

```diff
@@ -53,10 +53,10 @@
    * @param mount mount point
    * @return boolean
    **/
-    boolean check_raid_mount_points(string mount)
+    boolean check_raid_mount_points(string mount, string raid_level)
     {
         list<string> not_allowed_raid_mount_points  = [];
-	if (Arch::ppc() || Arch::s390())
+	if ((Arch::ppc() && raid_level != "raid1") || Arch::s390())
```

Comment:
Please add to comment explanation why ppc allows raid1 and s390 not, ideally link to bug or fate entry.